docs2/site/docs/guides/migration.md

```diff
@@ -69,7 +69,7 @@ The `DateGraphType` has been split into multiple types.  [See the GitHub issue](
 
 Fields, enumerations, and arguments all now have their names validated according to the GraphQL spec, which is `/[_A-Za-z][_0-9A-Za-z]*/`.
 
-`QueryArgument` names are now run through the `IFieldNameConverter` set on the `Schema`.
+`QueryArgument` names are now run through the `INameConverter` set on the `Schema`.
```

Comment:
This should not be changed since this is the upgrade guide from 0.17 to 2.x.  This should be put in an upgrade guide from 2.x to 3.x.

Reply:
Done.  Sorry about that!

Reply:
My apologies, I also did not notice.